src/table/src/metadata.rs

```diff
@@ -140,6 +141,13 @@ impl TableMetaBuilder {
     }
 }
 
+struct SplitResult<'a> {
+    columns_at_first: Vec<&'a AddColumnRequest>,
+    columns_location_map: HashMap<String, Vec<&'a AddColumnRequest>>,
+    columns_at_last: Vec<&'a AddColumnRequest>,
+    column_names: Vec<String>,
```

Comment:
Is this field needed? Don't see where it's been used.

Reply:
It's used at https://github.com/lizhemingi/greptimedb/blob/support-first-after/src/table/src/metadata.rs#L301

Just for log and the same as before.